src/Interpreters/InterpreterShowColumnsQuery.cpp

```diff
@@ -24,7 +24,10 @@ String InterpreterShowColumnsQuery::getRewrittenQuery()
 {
     const auto & query = query_ptr->as<ASTShowColumnsQuery &>();
 
-    const bool use_mysql_types = getContext()->getSettingsRef().use_mysql_types_in_show_columns;
+    const auto & settings = getContext()->getSettingsRef();
+    const bool use_mysql_types = settings.use_mysql_types_in_show_columns;
+    const bool remap_string_as_text = settings.mysql_map_string_to_text_in_show_columns;
```

Comment:
`remap_` --> `map_`

Reply:
Let's change this in one of the follow-ups.

Reply:
Sure, please merge then once all test ran.

Reply:
@rschu1ze, I don't seem to have the necessary permissions to merge.

Reply:
Okay, I can do it.